lib/receipt.js

```diff
@@ -226,6 +226,12 @@ class Receipt {
 }
 
 function hashReceipt(serializedReceipt) {
+    // Due to how Solidity encodes and hashes uint8 values, we must explicitly
+    // define them as uint8 values to avoid Solidity encoding and hashing them
+    // as uint256 values
+    serializedReceipt.seals.wallet.signature.v = {
+        type: 'uint8', value: serializedReceipt.seals.wallet.signature.v
```

Comment:
Wouldnt it be better to just serialize the "v" parameter as a string always, and thus avoid this whole "hack"?

Reply:
From my understanding that would not be possible currently. 

Maybe it would be possible to do some kind of transformation in Solidity to convert uint8 values to string, but maybe that would result in higher gas costs and complexity in the contracts, so not a desired solution.